_posts/2022-03-16-encodecall-bug.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+published: true
+title: 'abi.encodeCall Bug'
+date: '2022-03-16'
+author: Solidity Team
+category: Security Alerts
+---
+
+On March 10th, 2022 the Solidity team discovered a bug in the implementation of
+``abi.encodeCall`` when used together with fixed-length bytes literals.
+
+It was introduced together with ``abi.encodeCall`` in Solidity 0.8.11 and is fixed in 0.8.13.
+
+We assigned the bug a severity of "very low".
+
+## Which Contracts are Affected?
+
+You might be affected if you use ``abi.encodeCall(f, (...))`` where ``f`` takes a
+``bytesNN`` parameter and you provide the value for that parameter either as a
+hex literal (``0x1234`` or ``hex"abcd"``), as an explicit zero (``0``) or
```

Comment:
Explicit zeroes should be unaffected, since alignment doesn't matter.

Reply:
haha good point! :)